src/main/java/carpet/script/value/EntityValue.java

```diff
@@ -1668,12 +1690,57 @@ else if (v instanceof ListValue)
             }
         });
 
+        put("blue_skull",(e,v)->{
+            if (e instanceof WitherSkull w){
+                w.setDangerous(v.getBoolean());
+            }
+            
+        });
+        put("offering_flower",(e,v)->{
+            if (e instanceof IronGolem ig){
+                ig.offerFlower(v.getBoolean());
+            }
+            
+        });
+        put("item", (e, v) -> {
+                ItemStack item=getItemStackFromValue(v).copy();
+                if(e instanceof ItemEntity iteme)            
+                    iteme.setItem(item);
+                if(e instanceof ItemFrame iteme)
+                    iteme.setItem(item);  
+        });
         // "dimension"      []
-        // "item"           []
         // "count",         []
         // "effect_"name    []
     }};
 
+    static public ItemStack getItemStackFromValue(Value value){
+        List<Value> items;
+        if(value instanceof ListValue blv && blv.length()==3){
+            items = blv.getItems();
+        }else if(value.isNull()){
+            return ItemStack.EMPTY;
+        }else{
+            items= List.of(StringValue.of(value.getString()),Value.ONE,Value.NULL);
+        }
+        String id=items.get(0).getString();
+        int count=((NumericValue)items.get(1)).getInt();
+        CompoundTag nbt=null;
+        if (!items.get(2).isNull()){
+            if(items.get(2) instanceof NBTSerializableValue nbtv){
+                nbt=nbtv.getCompoundTag();
+            }
+            else{
+                nbt=new NBTSerializableValue(items.get(2).getString()).getCompoundTag();
+            }
+        }
+        try{
+            return NBTSerializableValue.parseItem(id,nbt).createItemStack(count,false);
+        } catch (CommandSyntaxException e1) {
+            e1.printStackTrace();
```

Comment:
Don't do `printStackTrace`. Instead, if the exception is expected to happen for some reason, handle it. Else, pass it through `CarpetSettings.LOG` with a message saying what was going on and possibly throw an exception unless it's not that bad.

Btw why is a `CommandSyntaxException` possible here? Seems like you may be calling something to do with commands? Though idk, as mentioned I haven't really checked this method yet.